tutoraspects/templates/aspects/build/aspects-superset/requirements.txt

```diff
@@ -1,3 +1,4 @@
 authlib  # OAuth2
 clickhouse-connect>0.5,<0.6
 sentry-sdk[flask]
+urllib3>=1.26.15,<2
```

Comment:
`urlib3==2.0.0` introduced a change that broke `selenium`. For this reason is pinned